db/crud.go

```diff
@@ -2653,6 +2656,11 @@ func (db *DatabaseCollectionWithUser) updateAndReturnDoc(ctx context.Context, do
 					updatedDoc.XattrsToDelete = append(updatedDoc.XattrsToDelete, base.GlobalXattrName)
 				}
 			}
+			if docUpdateEvent == NoHLVUpdateForTest {
+				// If this is a test update where we don't want to update the HLV, we should remote VV xattr here
+				// this will simulate a pre upgraded document being cerated in the bucket
```

Comment:
spelling mistake

```suggestion
				// This is a test seam to allow CRUD operations to push non HLV aware documents
				// this will simulate a pre upgraded documents
```